ansible/plugins/inventory/nodejs_yaml.py

```diff
@@ -23,35 +23,32 @@
 #
 
 from __future__ import print_function
+
 import argparse
-try:
-    import configparser
-except ImportError:
-    import ConfigParser as configparser
-try:
-    from future_builtins import filter # Python 2
-except ImportError:
-    pass # Python 3
 import json
-import yaml
 import os
-import sys
 import subprocess
+import sys
 
+import yaml
+try:
+    import configparser
+except ImportError:
+    import ConfigParser as configparser
 
 valid = {
-  # taken from nodejs/node.git: ./configure
-  'arch': ('armv6l', 'armv7l', 'arm64', 'ia32', 'mips', 'mipsel', 'ppc',
-           'ppc64', 'x32', 'x64', 'x86', 's390', 's390x'),
-
-  # valid roles - add as necessary
-  'type': ('infra', 'release', 'test'),
-
-  # providers - validated for consistency
-  'provider': ('azure', 'digitalocean', 'joyent', 'ibm', 'linuxonecc',
-               'macstadium', 'marist', 'mininodes', 'msft', 'osuosl',
-               'rackspace', 'requireio', 'scaleway', 'softlayer', 'voxer',
-               'packetnet', 'nearform')
+    # taken from nodejs/node.git: ./configure
+    'arch': ('arm', 'arm64', 'ia32', 'mips', 'mipsel', 'mips64el', 'ppc',
```

Comment:
you added arm and x86_64? that doesn't seem related to py3 compat

Reply:
Reverted